src/pages/Registration/Steps/getRegistrationState.ts

```diff
@@ -63,9 +63,13 @@ export function getRegistrationState(reg: SavedRegistration): {
 
   if (isDoneDocs(reg)) {
     const { ContactPerson: c, Registration: r } = reg;
+    const isSignedFSA =
```

Comment:
nit:
```suggestion
    const isSignedFSA = r.Documentation.DocType !== "FSA" || !!r.Documentation.SignedFiscalSponsorshipAgreement;
```

Reply:
saves a few character of code but I can't understant it the first time I read it 

original:
FSA is signed  - `DocType` is `FSA` ? check if FSA is signed , otherwise no need to check

suggested:
FSA is signed - `DocType` is `not` `FSA` `or` `FSA` is signed

I also noticed you changed a lot of these type constructs on your previous PR, which I believe is not necessary

Reply:
It's just a suggestion, you don't have to implement it.

> I also noticed you changed a lot of these type constructs on your previous PR, which I believe is not necessary

I'd agree, in retrospect maybe it wasn't necessary. Had you or Andrey pointed this out, I've would've removed those changes from the refactor PR